README.md

```diff
@@ -37,3 +37,187 @@ Following are some of the key features of *eProsima DDS Router*:
+example, on Ubuntu use the command:
+
+```batch
+sudo apt install cmake g++ pip wget git
+```
+
+#### Colcon
+
+[colcon](https://colcon.readthedocs.io/en/released/) is a command line tool based on [CMake](https://cmake.org/) aimed at building sets of software packages. Install the ROS 2 development tools ([colcon](https://colcon.readthedocs.io/en/released/) and [vcstool](https://pypi.org/project/vcstool/)) by executing the following command:
+
+```batch
+pip3 install -U colcon-common-extensions vcstool
+```
+
+If this fails due to an Environment Error, add the `--user` flag to the `pip3` installation command.
+
+#### Gtest
+
+[Gtest](https://github.com/google/googletest) is a unit testing library for C++. By default, *DDS Router* does not
+compile tests. It is possible to activate them with the opportune [CMake options](https://colcon.readthedocs.io/en/released/reference/verb/build.html#cmake-options) when calling [colcon](https://colcon.readthedocs.io/en/released/) or
+[CMake](https://cmake.org/). For a detailed description of the Gtest installation process, please refer to the
+[Gtest Installation Guide](https://github.com/google/googletest).
+
+### Dependencies
+
+#### Asio and TinyXML2 libraries
+
+Asio is a cross-platform C++ library for network and low-level I/O programming, which provides a consistent asynchronous model. TinyXML2 is a simple, small and efficient C++ XML parser. Install these libraries using the package manager of
+the appropriate Linux distribution. For example, on Ubuntu use the command:
+
+```batch
+sudo apt install libasio-dev libtinyxml2-dev
+```
+
+#### OpenSSL
+
+[OpenSSL](https://www.openssl.org/) is a robust toolkit for the TLS and SSL protocols and a general-purpose cryptography library. Install OpenSSL using the package manager of the appropriate Linux distribution. For example, on Ubuntu use the command:
+
+```batch
+sudo apt install libssl-dev
+```
+
+#### eProsima dependencies
+
+If it already exists in the system an installation of *Fast DDS* library with version greater than *2.4.0*, just source
+this library when building the *DDS Router* application by using the command:
+
+```batch
+source <fastdds-installation-path>/install/setup.bash
+```
+
+In other case, just download *Fast DDS* project from sources and build it together with *DDS Router* using colcon as it
+is explained in the following section.
+
+### Colcon installation
+
+1. Create a `DDS-Router` directory and download the `.repos` file that will be used to install *DDS Router* and its dependencies:
+
+```batch
+mkdir -p ~/DDS-Router/src
+cd ~/DDS-Router
+wget https://raw.githubusercontent.com/eProsima/DDS-Router/main/ddsrouter.repos
+vcs import src < ddsrouter.repos
+```
+
+2. Build the packages:
+
+```batch
+colcon build
+```
+
+### CMake installation
```

Comment:
NIT: I would not add this section. I do not think people use it, and If someone is interested, could go to the docu. I would do this Readme as a very easy and simple way to install, and not add extra possibilities